api/system-backup/read

```diff
@@ -104,6 +104,28 @@ sub find_latest_log
     return ($latest_filename, $latest_ts);
 }
```

Comment:
I think we can totally remove this code along with `last-log` action: it seems the UI doesn't use it anymore.
If the UI still use it, please merge `find_all_logs and `find_latest_log` in one function where `find_all_logs` return an ordered array.

Reply:
I think it's a good pattern to never remove an API, because someone can use it in an external way (scripts, etc...). UI doesn't need anymore the API but I would like to keep it. However, we can merge it in one function to clean the code